src/helpers/getDelayType.js

```diff
@@ -4,7 +4,7 @@ function getDelayType(delay) {
     return "";
   }
 
-  return delay >= 10 ? "early" : delay <= -60 * 3 ? "late" : "on-time";
+  return delay <= -10 ? "early" : delay >= 60 * 3 ? "late" : "on-time";
```

Comment:
I see you've fixed this later, but what is the reason to this change in the first place?